packages/api-server/src/index.ts

```diff
@@ -11,15 +11,29 @@ if (process.argv0 === 'api-server') {
   console.log()
 }
 
-const { port, socket } = yargs
+const { port, socket, rootPath } = yargs
   .option('port', { default: 8911, type: 'number', alias: 'p' })
-  .option('socket', { type: 'string' }).argv
+  .option('socket', { type: 'string' })
+  .option('root-path', {
+    default: '/',
+    type: 'string',
+    desc: 'Root path where your api functions are served',
+  })
+  .coerce('root-path', (path) => {
+    // Make sure that we create a root path that start and ending slash (/)
+    const prefix = path.charAt(0) !== '/' ? '/' : ''
+    const suffix = path.charAt(path.length - 1) !== '/' ? '/' : ''
+
+    return `${prefix}${path}${suffix}`
+  }).argv
 
 http({ port, socket }).on('listening', () => {
   if (socket) {
     console.log(`Listening on ${socket}`)
   } else {
-    console.log(`Listening on http://localhost:${port}`)
+    console.log(`Listening on http://localhost:${port}${rootPath}`)
   }
   console.log()
 })
+
+export { rootPath }
```

Comment:
@peterp I'm not entirely sure how I would pickup `rootPath` in `http.ts` unless I export and import it. Thoughts?

Reply:
Oh, I would explicitly pass it into `http({ port, socket, rootPath })`

Reply:
You are absolutely right. Not sure how my thought process got me to exporting it from the beginning. Sorry. 😅
I'll align this by the end of the day.